third_party/openvino/ienn/src/ie_nn_c_api.h

```diff
@@ -96,25 +96,42 @@ enum ie_auto_pad : uint32_t {
   SameLower = 0x00000002,
 };
 
+enum ie_activation_type : uint32_t {
+  None = 0x00000000,
+  Clamp = 0x00000001,
+  Relu = 0x00000002,
+  Sigmoid = 0x00000003,
+  LeakyRelu = 0x00000004,
+};
+
+typedef struct ie_constant {
+  const float* value = nullptr;
+  const int32_t* dimensions;
+  uint32_t dimensionsCount = 0;
+} ie_constant_t;
+
+typedef struct ie_activation {
```

Comment:
Move the implementation of fused operator to `GraphIE `to reduce those struct (`ie_activation`, `ie_constant`, `ie_activation_type`),  for example add relu in Cov2d, you can call `IE(ie_model_add_relu)(mIeModel, &input, &ieOperand)`. so you need to wrap a common function like [AddActivationNode ](https://github.com/webmachinelearning/webnn-native/pull/62/files#diff-57a5f3417f110f4c523fba96c49ec775b7171f808cfca01c1c16c178bbb5bfeaR216)in  `GraphIE `

Reply:
OK, thanks! I will update as this.